PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,20 @@
-All Submissions:
+## For External Submissions:
 
-* [ ] Have you followed the guidelines in our Contributing document?
-* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
+* [ ] Have you followed the guidelines in our [Contributing](https://github.com/luno/.github/blob/main/CONTRIBUTING.md) file?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
 
 <!-- You can erase any parts of this template not applicable to your Pull Request. -->
-### Checklist before requesting a review
+## Checklist before requesting a review
+- [ ] If it's a non-trivial change, I've created a new [issue](../issues) first
 - [ ] I have performed a self-review of my code
 - [ ] I have added thorough tests, if appropriate.
 
-### New Feature Submissions:
+## New Feature Submissions:
 
 1. [ ] Does your submission pass tests?
 2. [ ] Did you install pre-commit hooks before committing?
```

Comment:
Can we remove this line now if precommit hooks aren't needed anymore?

Reply:
Yep good point, have removed. And simplified in general 👍